runtimes/java/catala/runtime/exception/CatalaException.java

```diff
@@ -0,0 +1,16 @@
+package catala.runtime.exception;
+
+// I guess catala exceptions should be unchecked?
+// (hence the RuntimeException base)
+// Rationale: I'm not sure an attempt to recover from a 
```

Comment:
You would want to catch from a `CatalaException` outside of the generated Catala code, i.e in the application calling the Catala code. However, if by "recovering" you mean resuming the Catala computation with a continuation, then no we don't want that.